website/client/components/challenges/myChallenges.vue

```diff
@@ -126,8 +126,14 @@ export default {
       let filters = this.filters;
```

Comment:
since we have filters defined here as a var, we should use it all the times or never (and instead use `this.filters`), both are okay but we should stick to one :)